clients/drcachesim/tracer/instru.h

```diff
@@ -277,6 +282,7 @@ class instru_t {
     bool memref_needs_full_info_;
     drvector_t *reg_vector_;
     bool disable_optimizations_;
+    uint64 frozen_timestamp_ = 0;
```

Comment:
Maybe add a comment here on this field as well.

Reply:
Done.